.github/workflows/cypress-workflow.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - cypress
```

Comment:
Assuming this was for testing on the cypress branch, can remove this from main

Reply:
actually I meant to leave this, but yeah, it makes sense to remove this from main. removed